src/shared/components/case-editor/case-edit-form/case-edit-form.html

```diff
@@ -1,4 +1,5 @@
-<ng-container ccdConditionalShowForm [formGroup]="formGroup" [caseFields]="fields" [contextFields]="caseFields" *ngFor="let field of fields">
+<ng-container ccdConditionalShowForm [formGroup]="formGroup" [caseFields]="fields" [contextFields]="caseFields"
```

Comment:
Any reason for this? Prettier being naughty again?

Reply:
Overflowing lines